bin/ResumeChecker.py

```diff
@@ -227,6 +227,41 @@ def term_match(string_to_search, term):
                       str(string_to_search) + ': ' + str(exception_instance))
         return None
 
+def check_title_city_state(string_to_search):
+    """
+        Find the title of the job, location by city and state
+        :param string_to_search: A string to check for a physical address in
+        :type string_to_search: str
+        :return: array of strings with job, location by city and state
+        :rtype: bytearray
```

Comment:
byte array is technically correct, but `:rtype: str` would be more consistent with the `term_match()` method signature.